janitor/functions/conditional_join.py

```diff
@@ -314,15 +278,12 @@ def _conditional_join_preliminary_checks(
 def _conditional_join_type_check(
     left_column: pd.Series, right_column: pd.Series, op: str
 ) -> None:
-    """
-    Raise error if column type is not
-    any of numeric or datetime or string.
-    """
+    "Raise error if column type is not"
+    "any of numeric or datetime or string."
```

Comment:
```suggestion
    """
    Raise error if column type is not any of numeric or datetime or string.
    """
```
Let's use multiline string here instead